locales/id/id.json

```diff
@@ -431,18 +431,18 @@
     "Restore Selected": "Pulihkan Yang Dipilih",
     "results": "hasil",
     "Reunion": "Reuni",
-    "Role": "Peran",
+    "Role": "Tugas",
```

Comment:
In this one, "Peran" is more fit than "Tugas". "Tugas" could also mean "Task" while the only translation of "Peran" in English is "Role"